apps/aecontract/src/aect_contracts.erl

```diff
@@ -129,18 +129,23 @@ is_legal_serialization_at_height(?ABI_SOPHIA_1, Vsn, Height) ->
 is_legal_version_in_protocol(create, #{vm := ?VM_AEVM_SOPHIA_1, abi := ?ABI_SOPHIA_1}, ProtocolVersion) ->
     case ProtocolVersion of
         ?ROMA_PROTOCOL_VSN    -> true;
-        ?MINERVA_PROTOCOL_VSN -> false
+        ?MINERVA_PROTOCOL_VSN -> false;
+        ?FORTUNA_PROTOCOL_VSN -> false
     end;
 is_legal_version_in_protocol(create, #{vm := ?VM_AEVM_SOPHIA_2, abi := ?ABI_SOPHIA_1}, ProtocolVersion) ->
     case ProtocolVersion of
         ?ROMA_PROTOCOL_VSN    -> false;
-        ?MINERVA_PROTOCOL_VSN -> true
+        ?MINERVA_PROTOCOL_VSN -> true;
+        ?FORTUNA_PROTOCOL_VSN -> true %% TODO: Revise this before release
     end;
 is_legal_version_in_protocol(call, #{vm := VMVersion}, ProtocolVersion) ->
     case ProtocolVersion of
         ?ROMA_PROTOCOL_VSN    when VMVersion =:= ?VM_AEVM_SOPHIA_1 -> true;
         ?MINERVA_PROTOCOL_VSN when VMVersion =:= ?VM_AEVM_SOPHIA_1;
                                    VMVersion =:= ?VM_AEVM_SOPHIA_2 -> true;
+        %% TODO: Revise this before release
+        ?FORTUNA_PROTOCOL_VSN when VMVersion =:= ?VM_AEVM_SOPHIA_1;
+                                   VMVersion =:= ?VM_AEVM_SOPHIA_2 -> true;
```

Comment:
And this!

One could perhaps do some `ifdef` trickery?!

Reply:
The protocol version comes from network id or "chain > hard_forks" config, so this seems ok to me.